blog/_posts/2017-08-01-one-month-of-RGSoC.md

```diff
@@ -0,0 +1,42 @@
+---
+title: One month of RGSoC
+layout: post
+created_at: Tue Aug 01 2017
+permalink: blog/2017-08-01-one-month-of-RGSoC
+current: blog
+author: Vaishali Thakkar
+twitter: RailsGirlsSoC
```

Comment:
we somehow always use the generic RailsGirlsSoC twitter handle here, I don't know why really — but feel free to replace with your personal one if you prefer :)